Code/BasicFilters/json/SLICImageFilter.json

```diff
@@ -0,0 +1,145 @@
+{
+  "name" : "SLICImageFilter",
+  "template_code_filename" : "ImageFilter",
+  "template_test_filename" : "ImageFilter",
+  "number_of_inputs" : 1,
+  "pixel_types" : "typelist::Append<BasicPixelIDTypeList, VectorPixelIDTypeList>::Type",
+  "output_pixel_type" : "uint32_t",
+  "members" : [
+    {
+      "name" : "SuperGridSize",
+      "type" : "unsigned int",
+      "default" : "std::vector<unsigned int>(3, 50)",
+      "dim_vec" : 1,
+      "doc" : "",
+      "itk_type" : "typename FilterType::SuperGridSizeType",
+      "briefdescriptionSet" : "The expected superpixel size and shape.",
+      "detaileddescriptionSet" : "The requested size of a superpixel used to form a regular grid for initialization and limits the search space for pixels. The size may be set anisotropically to provide a directional bias. This may be set to reflect spacing of this image.",
+      "briefdescriptionGet" : "The expected superpixel size and shape.",
+      "detaileddescriptionGet" : "The requested size of a superpixel used to form a regular grid for initialization and limits the search space for pixels. The size may be set anisotropically to provide a directional bias. This may be set to reflect spacing of this image."
+    },
+    {
+      "name" : "SpatialProximityWeight",
+      "type" : "double",
+      "default" : "10.0",
+      "briefdescriptionSet" : "The spatial weight for the distance function.",
+      "detaileddescriptionSet" : "Increasing this value makes the superpixel shape more regular, but more varied in image values. The range of the pixel values and image dimension can effect the appropriate value.",
```

Comment:
Possibly say that the default value is appropriate for 2D images in the CIE-Lab color space?